community/portworx/values.yaml

```diff
@@ -1,5 +1,8 @@
 # Please uncomment and specify values for these options as per your requirements.
 
+generateSTCForMigration: true         # Only used for daemonset to operator migration.
```

Comment:
Do we need to enable the migration by default?

Reply:
it's enabled by default.